Mathlib/Topology/UniformSpace/AbstractCompletion.lean

```diff
@@ -66,7 +66,7 @@ structure AbstractCompletion (α : Type u) [UniformSpace α] where
   /-- The completion is a T₀ space. -/
   separation : T0Space space
   /-- The map into the completion is uniform-inducing. -/
-  uniformInducing : UniformInducing coe
+  isUniformInducing : IsUniformInducing coe
```

Comment:
I realize this is a structure field. Do we provide deprecations for these? It seems less necessary because Lean should normally be able to tell the user what the problem is.

Reply:
Doesn't cost much to add it, so I've added it.

cc @adomani if you want to modify the script to account for structure field renames